src/cclib/io/cjsonwriter.py

```diff
@@ -327,7 +327,8 @@ class NumpyAwareJSONEncoder(json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, np.ndarray):
             if obj.ndim == 1:
-                return obj.tolist()
+                nan_list = obj.tolist()
```

Comment:
But wouldn't that be equivalent to removing all the NaN values and in turn removing all the 'null' types from the attribute in the Json?

Presently we are passing the attributes as is, and the null values indicates the incomplete calculation